### ISSUE TITLE
Use Lora with hosted inference API

### ISSUE THREAD

**Issue by lionel-alves** [huggingface/api-inference-community#281]
Hi,

I would like to choose a model and a lora and generate images with it using the hosted inference API
As showed here: https://huggingface.co/pcuenq/pokemon-lora

If parameters has a lora_model_id use merge with the lora model.

**Comment by pcuenca** [comment]
> If you use a diffusers LoRA I think this is already what will happen, the lora will download the mother model and use them.

Yes, exactly! The mother model is referenced in the `base_model` metadata field as shown here: https://huggingface.co/pcuenq/pokemon-lora/blob/main/README.md?code=true#L4
